src/lib/xiaomi.ts

```diff
@@ -1381,6 +1375,43 @@ export const xiaomiModernExtend = {
         attribute: 'presentValue',
         ...args,
     }),
+    aqaraVoc: (args?: Partial<modernExtend.NumericArgs>) => modernExtend.numeric({
+        name: 'voc',
+        cluster: 'genAnalogInput',
+        attribute: 'presentValue',
+        configureReporting: {minimumReportInterval: 10, maximumReportInterval: constants.repInterval.HOUR, reportableChange: 5},
+        configureEndpointId: 1,
+        description: 'Measured VOC value',
+        unit: 'ppb',
+        readOnly: true,
+        ...args,
+    }),
+    aqaraAirQuality: (args?: Partial<modernExtend.EnumLookupArgs>) => modernExtend.enumLookup({
+        name: 'air_quality',
+        lookup: {'excellent': 1, 'good': 2, 'moderate': 3, 'poor': 4, 'unhealthy': 5},
+        cluster: 'aqaraOpple',
+        attribute: 'airQuality',
+        configureEndpointId: 1,
```

Comment:
Since we still call setupAttributes() but in readOnly when there is no configureReporting payload we still need to set it for these devices to avoid the error of it not finding aqaraOpple cluster.

Perhaps we should name this `endpointId` instead ?

Reply:
Let's call it `endpointID` indeed